### PR TITLE
Fix most instances of D404 ("docstring should not start with 'this'").

### PR DESCRIPTION
**Pull request by anntzer**
... and associated rewordings.

(xref https://github.com/matplotlib/matplotlib/pull/17145#issuecomment-613906680).

## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: lib/matplotlib/axes/_base.py [hunk 1]
```diff
@@ -2774,9 +2775,11 @@ def draw_artist(self, a):
 
     def redraw_in_frame(self):
         """
-        This method can only be used after an initial draw which
-        caches the renderer.  It is used to efficiently update Axes
-        data (axis ticks, labels, etc are not updated).
+        It is used to efficiently redraw Axes data (axis ticks, labels, etc.,
```
Comment (timhoffm) [inline lib/matplotlib/axes/_base.py:2778]:
```suggestion
        Efficiently redraw Axes data (axis ticks, labels, etc.,
```

File: lib/matplotlib/backends/backend_wx.py [hunk 2]
```diff
@@ -1026,9 +1026,9 @@ def Destroy(self, *args, **kwargs):
 
 class FigureManagerWx(FigureManagerBase):
     """
-    This class contains the FigureCanvas and GUI frame
+    A class contains the FigureCanvas and GUI frame.
```
Comment (timhoffm) [inline lib/matplotlib/backends/backend_wx.py:1029]:
```suggestion
    Container/controller for the FigureCanvas and the GUI frame.
```

File: lib/matplotlib/quiver.py [hunk 3]
```diff
@@ -1013,48 +1013,48 @@ def _find_tails(self, mag, rounding=True, half=5, full=10, flag=50):
     def _make_barbs(self, u, v, nflags, nbarbs, half_barb, empty_flag, length,
                     pivot, sizes, fill_empty, flip):
         """
-        This function actually creates the wind barbs.  *u* and *v*
-        are components of the vector in the *x* and *y* directions,
-        respectively.
+        Actually creates the wind barbs.
```
Comment (timhoffm) [inline lib/matplotlib/quiver.py:1016]:
```suggestion
        Create wind barbs.
```

File: lib/matplotlib/tight_bbox.py [hunk 4]
```diff
@@ -1,5 +1,5 @@
 """
-This module is to support the *bbox_inches* parameter in `.Figure.savefig`.
+Support module for the *bbox_inches* parameter in `.Figure.savefig`.
```
Comment (timhoffm) [inline lib/matplotlib/tight_bbox.py:2]:
```suggestion
Helper module for the *bbox_inches* parameter in `.Figure.savefig`.
```
I have the impression that "helper module" is more common than "support module", but not sure.

File: lib/matplotlib/transforms.py [hunk 5]
```diff
@@ -211,7 +211,7 @@ def frozen(self):
 
 class BboxBase(TransformNode):
     """
-    This is the base class of all bounding boxes, and provides read-only access
+    The base class of all bounding boxes, which provides read-only access
     to its data.  A mutable bounding box is provided by the `Bbox` class.
```
Comment (timhoffm) [inline lib/matplotlib/transforms.py:215]:
```suggestion
    The base class of all bounding boxes.
    
    This box is immutable. See `Bbox` for a mutable bounding box.
```

**Comment by anntzer** [comment]
thanks, all handled.

**Pull request merged**